core/src/validator.rs

```diff
@@ -2268,7 +2268,7 @@ mod tests {
             sender.send(()).unwrap();
         });
 
-        let timeout = Duration::from_secs(60);
+        let timeout = Duration::from_secs(120);
```

Comment:
was this bumped in the master pr too?

please don't address ci issues in unrelated PRs. we'll never fix them if they're hidden like this

Reply:
On master pr it was bumped from 30 to 60.
But for whatever reason it started failing here in backport so I bumped it further up to 120.

Agree about that this should have been addressed separately but it is urgency/priority trade-off here as you know. It is also a pretty innocuous test-only change. Bumping the timeout might as well be the right thing to do here.

Reply:
This initially passed and CI was green:
https://buildkite.com/solana-labs/solana/builds?branch=pull%2F30832%2Fhead
but I had to resolve the merge conflicts before merge. But then the CI kept failing, so :man_shrugging: